### PR TITLE
Restore unbound::local_zone function

### PR DESCRIPTION
**Pull request by jlutran**
This PR fixes issue #177

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,6 +188,13 @@ class {'unbound:'
 }
 ```
 
+Or, using the legacy way
```
Comment (xaque208) [inline README.md]: It doesn't have to be legacy.

File: manifests/local_zone.pp [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# == Parameters:
+#
+# [*local_zone*]
+#   (required) Unbound::Local_zone custom type
+#
+# [*config_file*]
+#   (optional) name of configuration file
+#
+define unbound::local_zone (
+  Unbound::Local_zone $local_zone,
```
Comment (xaque208) [inline manifests/local_zone.pp]: I think the type here needs to be something other than the name of the class.

**Comment by xaque208** [comment]
Thanks for sending this in @jlutran, and apologies for causing trouble in your environment.  We might need to rename either the type or the defined type for this to work as is.

**Comment by jlutran** [comment]
Ok, I changed ```unbound::local_zone```function to use ```Unbound::Local_zone_type``` type only.
This means we won't be able to pass a ```Unbound::Local_zone_type``` hash to configure multiple local zones at the same time but I don't think it's a big deal since use can use the new ```local_zone``` parameter from ```unbound``` class to do it now.

**Comment by xaque208** [comment]
This looks good to me.  Thanks for the contribution.

**Pull request merged**